tests/vm_console_proxy_test.go

```diff
@@ -0,0 +1,188 @@
+package tests
+
+import (
+	"reflect"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+	ssp "kubevirt.io/ssp-operator/api/v1beta1"
+	vm_console_proxy "kubevirt.io/ssp-operator/internal/operands/vm-console-proxy"
+)
+
+var _ = Describe("VM Console Proxy Operand", func() {
+	var (
+		clusterRoleResource        testResource
+		clusterRoleBindingResource testResource
+		serviceAccountResource     testResource
+		serviceResource            testResource
+		deploymentResource         testResource
+		configMapResource          testResource
+	)
+
+	BeforeEach(func() {
+		strategy.SkipSspUpdateTestsIfNeeded()
+
+		updateSsp(func(foundSsp *ssp.SSP) {
+			if foundSsp.GetAnnotations() == nil {
+				foundSsp.Annotations = make(map[string]string)
+				foundSsp.Annotations[vm_console_proxy.EnableAnnotation] = "true"
+			} else {
+				foundSsp.Annotations[vm_console_proxy.EnableAnnotation] = "true"
```

Comment:
This line can be moved out of the `if` statement, because it is in both brnaches.

Can you also add the `VmConsoleProxyNamespaceAnnotation` here?

Reply:
Good point, thanks!